apps/api/src/app/user/dtos/register-user.dto.ts

```diff
@@ -0,0 +1,26 @@
+import {ApiProperty} from "@nestjs/swagger";
+import {UserRole} from "../models/user-roles";
```

Comment:
**issue:** take care to format all of your files with prettier. Please select the `apps/api` folder and do a CTRL+ALT+L (take care you set the IDE to use prettier)

In your Webstorm you can set it to always reformat it after a save (File/Settings=>search prettier => reformat on save and format => apply)